hello-world-notification-microservice/demo/measurements.py

```diff
@@ -0,0 +1,101 @@
+import base64
```

Comment:
Should we name this file something other than measurements.py (as it does more than send measurements)? Maybe demo.py or device.py